packages/front-end/pages/experiment/[eid].tsx

```diff
@@ -754,6 +763,20 @@ const ExperimentPage = (): ReactElement => {
                   </div>
                 </RightRailSection>
               )}
+              <hr />
+              <RightRailSection title="Watcher List">
+                <RightRailSectionGroup
+                  title={`Watchers - ${allUserIds?.length}`}
+                  empty=""
+                ></RightRailSectionGroup>
+                {allUserIds?.map((id) => (
+                  <ul key={id}>
+                    <li className="bg-light p-0.5">
+                      {users.has(id) ? users.get(id).name : id}
+                    </li>
+                  </ul>
+                ))}
+              </RightRailSection>
```

Comment:
There's a built-in way to render a list for the right rail group:

```suggestion
              <RightRailSection title="Watcher List">
                <RightRailSectionGroup
                  title={`Watchers - ${allUserIds?.length}`}
                  type="list"
                >
                  {allUserIds?.map((id) => ( users.get(id)?.name || id ))}
                </RightRailSectionGroup>
              </RightRailSection>
```